includes/ziti/ziti.h

```diff
@@ -689,12 +698,16 @@ extern int ziti_accept(ziti_connection clt, ziti_conn_cb cb, ziti_data_cb data_c
  * When no longer needed a [connection](#ziti_connection) should be closed to gracefully disconnect. This
  * function should be invoked after any status is returned which indicates an error situation.
  *
+ * This method ensures initiates disconnect(if needed) and release all associated resources.
```

Comment:
> This method ensures initiates disconnect
needs some new words at the least but really isn't it sort of stated in the lines above? i started to rework it but - then realized i don't really have the time to offer anything insightful just yet.... :(